docs/quickstart-aks.md

```diff
@@ -209,6 +214,7 @@ You should also ensure that the `Microsoft.Compute` and `Microsoft.Network` prov
       --set azure.tenantId=$AZURE_TENANT_ID \
       --set azure.clientId=$AZURE_CLIENT_ID \
       --set azure.clientSecret=$AZURE_CLIENT_SECRET
+      --set modules.minStability=PREVIEW
```

Comment:
Missing trailing `\` on line above.

Reply:
missing the ` on the PowerShell version too! fixed